src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/websocket/WebSocketConstants.java

```diff
@@ -102,9 +106,7 @@ public class WebSocketConstants {
             Arrays.asList("CloseReason", "Session"));
 
     public static final Set<String> RAW_WRAPPER_OBJS = new HashSet<>(
-            Arrays.asList("String", "Boolean", "Integer", "Long", "Double", "Float"));
-    public static final Set<String> WRAPPER_OBJS = RAW_WRAPPER_OBJS.stream().map(raw -> "java.lang.".concat(raw))
-            .collect(Collectors.toSet());
+            Arrays.asList(BOOLEAN, INTEGER, LONG, DOUBLE, FLOAT, CHARACTER, BYTE, SHORT, STRING_CLASS_LONG));
```

Comment:
I think this needs some explanation since this changes the value of the constant. There are more names in this list than before. Are you sure that these additional classes belong here? Did you test this? I think even if this is an appropriate change to make the best thing to do process wise would have been to open a separate GitHub issue which describes the problem and address it in its own PR.

Reply:
These types came from the Jakarta specification so it would be best to check there.

Reply:
@mrglavas, sorry the additional names were added by mistake; I’ll remove them. Aside from that, is this approach correct for addressing the issue?

Reply:
@anusreelakshmi934 Yes, this is generally the right approach, though please see my other comment about investigating why some constants for unqualified names are still in use.